### ISSUE TITLE
deno_terminal: ansi color does not be handled correctly in file

### ISSUE THREAD

**Issue by familyboat** [denoland/deno#26689]
Steps to reproduce:

```sh
deno install --log-level trace npm:zustand
```
Ansi color displays correctly in terminal.

But with this:
```
deno install --log-level trace npm:zustand &> log
```
Ansi color does not display correctly in file.

If I use npm cli, ansi color displays correctly in both situations.

BTW, my editor is vscode, and I do not install ansi-related extension.

**Comment by cu8code** [comment]
@bartlomieju I would love to work on this.

**Comment by bartlomieju** [comment]
@cu8code I'm not yet sure. @familyboat can you please provide more info what logs are being colored? Are these logs coming from `npm:zustand` package?

**Comment by familyboat** [comment]
Yes, colored information is from `deno cli`, not from `npm:zustand` package. I created a repository, and you can check the [instructions](https://github.com/familyboat/bug-deno-ansi-color/blob/main/README.md). If 
 I miss something, you can mention it. @bartlomieju @cu8code

**Comment by cu8code** [comment]
A quick node to anyone working on this,  nodejs handle this was by check if the current environment is a tty `process.env.isTTY`, then it disable the color.
